core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -776,8 +776,8 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 			{
 				if (\is_array($v['label']))
 				{
-					$label = $v['label'][0];
-					$title = sprintf($v['label'][1], $id);
+					$label = $v['label'][0] ?? $k;
+					$title = sprintf($v['label'][1] ?? $k, $id);
```

Comment:
fix for extension see: https://github.com/oveleon/contao-component-style-manager/pull/46/files#diff-bc2c37cb346d11ba0a90db5779f412b8d9f79de1ed6a2a58451fe22a6efaf989L33